src/runes.c

```diff
@@ -41,7 +41,16 @@ void DoDropRune(int rune, qbool s)
 	if (pos == NULL)
 	{
 		pos = self;
-		movetype = (int) cvar("k_ctf_rune_bounce") & 1 ? MOVETYPE_BOUNCE : MOVETYPE_TOSS;
+		if (on_respawn)
+		{
+			// Rune respawn, bounce adds randomization.
+			movetype = (int) cvar("k_ctf_rune_bounce") & 1 ? MOVETYPE_BOUNCE : MOVETYPE_TOSS;
```

Comment:
Could argue that this could be merged with the if condition, but decided to keep it separate for readability to have the two clear cases of on_respawn vs !on_respawn.